<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ignorepackagelist</class>
 <widget class="QDialog" name="ignorepackagelist">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>617</width>
    <height>368</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Qt-update-notifier ignore package window</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>601</width>
     <height>251</height>
    </rect>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::NoSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectItems</enum>
   </property>
   <column>
    <property name="text">
     <string>Package name</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="pbClose">
   <property name="geometry">
    <rect>
     <x>312</x>
     <y>330</y>
     <width>101</width>
     <height>33</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Close</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>300</y>
     <width>441</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>601</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Enter below a name of a package to be ignored</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pbAdd">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>330</y>
     <width>101</width>
     <height>33</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Add</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
